webbpsf/gridded_library.py

```diff
@@ -121,6 +130,8 @@ def __init__(self, instrument, filters="all", detectors="all", num_psfs=16, psf_
         # Set PSF attributes for the 3 kwargs that will be used before the calc_psf() call
         if "add_distortion" in kwargs:
             self.add_distortion = kwargs["add_distortion"]
+            if self.webb.name == "WFI":
```

Comment:
Minor: `if self.webb.name == "WFI"` seems awkward. We might want to consider renaming the `.webb` attribute to `.instrument` or something similar.

Reply:
That sounds fine to me! The only thing is that `self.instr` exists as an attribute for the instrument name, so I'm not sure if `instr` and `instrument` would be too similar (i.e. if may be better to then also change `self.instr` to something like `instr_name`). This change would be a pretty basic find/replace and then also double checking if the documentation needs to be updated